route.go

```diff
@@ -73,12 +75,38 @@ func (r *Route) GetError() error {
 	return r.err
 }
 
+// GetPath returns an string equal to the path of the route.
+func (r *Route) GetPath() string{
+	return r.regexp.path.template
+}
+
 // BuildOnly sets the route to never match: it is only used to build URLs.
 func (r *Route) BuildOnly() *Route {
 	r.buildOnly = true
 	return r
 }
 
+// MakePrivate sets the private flag making it not show up in GetRoutes.
+func (r *Route) MakePrivate() *Route {
```

Comment:
What is the goal of making a route private? I don't think this is worth adding to the public API, and it could be confused for some kind of authentication functionality.